### PR TITLE
[service.subtitles.subdivx] 0.3.5

### PR DESCRIPTION
**Pull request by ramiro**
### Description

Update to be in sync with latest subdivx.com search handling changes Also, add optional web search results caching.
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

### CODE REVIEW

**Comment by ramiro** [comment]
I've updated `addons.xml` to comply with new more strict checks performed by Travis CI although this addon is being submitted to the "gotham" branch/addons repo.

The only remaining ERROR is the existence of a `resources/language/English/strings.xml` file. But I've added (rather than replaced it with) a `string.po` file there so not tto  break installations on Kodi versions prior to Matrix.

Hopefully the PR can be accepted manually taking this in account.

Thanks!

**Comment by IvanRF** [comment]
@ramiro ahí subí el archivo `resources/language/English/strings.po` por si queres probar de nuevo

**Comment by basrieter** [comment]
> I've updated `addons.xml` to comply with new more strict checks performed by Travis CI although this addon is being submitted to the "gotham" branch/addons repo.
> 
> The only remaining ERROR is the existence of a `resources/language/English/strings.xml` file. But I've added (rather than replaced it with) a `string.po` file so not to break installations on Kodi versions prior to Matrix.
> 
> Hopefully the PR can be accepted manually taking this in account.
> 
> Thanks!

That is not needed. Gotham should be PO compatible. So please make Travis happy.

**Comment by ramiro** [comment]
> That is not needed. Gotham should be PO compatible. So please make Travis happy.

Done.  Thanks!

**Comment by IvanRF** [comment]
@basrieter could you check this PR? the plugin is currently failing and we need to update it. Thanks!

File: service.subtitles.subdivx/addon.xml [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
     <description lang="es">Busca y descarga subtítulos desde Subdivx.com</description>
     <description lang="en">Search and download subtitles from Subdivx.com</description>
-    <disclaimer lang="en"></disclaimer>
     <language>es en</language>
     <platform>all</platform>
     <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
-    <forum>http://forum.xbmc.org/forumdisplay.php?fid=143</forum>
-    <website></website>
+    <forum>https://forum.kodi.tv/forumdisplay.php?fid=143</forum>
     <email></email>
```
Comment (basrieter) [inline service.subtitles.subdivx/addon.xml:21]: Please remove the empty `<email>` tag

**Comment by IvanRF** [comment]
@ramiro queres hacer la pequeña corrección vos? así no hago lío con este commit 😃

**Pull request merged**